docs/source/traits_user_manual/defining.rst

```diff
@@ -460,6 +460,48 @@ on the class on which it was defined. For example::
     >>> # This is also OK, because mary's manager can be an Employee
     >>> mary.manager = fred
 
+.. index:: Either trait
+
+.. _either:
+
+Either
+::::::
+Another predefined trait that merit special explanation is Either. The
+Either trait is intended for attributes that may take a value of more than
+a single trait type, including None. The default value of Either is None.
```

Comment:
It would be worth noting that a different default can be provided with a `default=<whatever>` argument.